cocos/2d/CCCamera.h

```diff
@@ -128,9 +128,22 @@ class CC_DLL Camera :public Node
     const Mat4& getViewProjectionMatrix() const;
 
     /**
-    * Convert the specified point of viewport from screenspace coordinate into the worldspace coordinate.
-    */
-    void unproject(const Size& viewport, Vec3* src, Vec3* dst) const;
+     * convert the specified point of viewport from world-space coordinates into the screen-space coordinates.
+     *
+     * @param viewport The viewport size to use.
+     * @param src The world-space position.
+     * @param dst The screen-space position.
+     */
+    void project(const Size& viewport, const Vec3* src, Vec2* dst) const;
+    
+    /**
+     * Convert the specified point of viewport from screen-space coordinate into the world-space coordinate.
+     *
+     * @param viewport The viewport size to use.
+     * @param src The screen-space position.
+     * @param dst The world-space position.
+     */
+    void unproject(const Size& viewport, const Vec3* src, Vec3* dst) const;
     
```

Comment:
Can you do the same for the unproject? Probably have to deprecate the code method.

`Vec3 unproject(const Vec3 screenCoords)`